openssl-sys/src/lib.rs

```diff
@@ -1098,14 +719,30 @@ extern "C" {
                         -> c_int;
     pub fn PKCS12_free(p12: *mut PKCS12);
 
-    pub fn sk_free(st: *mut _STACK);
-    pub fn sk_pop_free(st: *mut _STACK, free: Option<unsafe extern "C" fn (*mut c_void)>);
-    pub fn sk_pop(st: *mut _STACK) -> *mut c_char;
+    #[cfg_attr(any(ossl101, ossl102), link_name = "sk_free")]
```

Comment:
How so? It's a bit of a weird attribute but allows us to work with OpenSSL 1.1.0 idioms while still working against older versions.